resources/views/help/developer.twig

```diff
@@ -0,0 +1,127 @@
+{% extends "layouts.master" %}
+
+{% block title %}Help for developers{% endblock %}
+
+{% block content %}
+<ol class="breadcrumb">
+    <li>{{ link_to_route('projects.index', 'Home') }}</li>
+    <li class="active">Help for developers</li>
+</ol>
+
+<h2>Help and guidance for developers</h2>
+
+<p>If you're completely new to localization under Haiku, please read the very basic blog <a href="https://www.haiku-os.org/community/getting-involved/translating/3rdparty">Localizing an application</a>. Also see the <a href="https://www.haiku-os.org/docs/api/group__locale.html">Haiku Book</a> and maybe search for examples in Haiku the tree with <a href="http://xref.plausible.coop/source/">OpenGrok</a>.</p>
+
+<p id="index">Quick links to the sections in this document:</p>
+<ul>
+	<li><a href="#overview">Overview</a></li>
+	<li><a href="#adding">Adding a project</a></li>
+	<li><a href="#uploading">Uploading existing xx.catkeys (translations)</a></li>
+	<li><a href="#updating">Updating en.catkeys</a></li>
+	<li><a href="#exporting">Exporting translated xx.catkeys</a></li>
+	<li><a href="#tips">Tips</a></li>
+	<li><a href="#tldr">TL;DR</a></li>
+</ul>
+
+<h2>
+<a href="#"><img src="images/up.png" style="border:none;float:right" alt="index" /></a>
```

Comment:
If you need a link use `{{ url('images/up.png') }}`. I'd also prefer if styles were in style.css.

Reply:
I'm not 100% positive I do that right. I added this to style.css:
```
img.centered {
	margin-bottom: 10px;
}

img.float {
	border: none; 
	float: right;
}
```
How would I assign the 'float' style to the "up" image (or the 'centered' to html_image('images/help/user-startpage.png')?

```<a href="#"><img class="float" src={{ url('images/up.png') }} alt="index" /></a>``` ?

Reply:
It would be even better if you used {{ html_image }} for up images as well.

Also change class names to be more descriptive, like *up* and *screenshot*.

See below:
```twig
{{ html_image('images/help/dev-upload-xx-catkeys.png', 'Uploading the en.catkeys', {'class': 'center-block'}) }}
```

Reply:
OK. Like that?
```<a href="#">{{ html_image('images/up.png', 'Go up', {'class': 'up'}) }}</a>```

Reply:
Exactly.